### PR TITLE
Fix grammar on en & en-us About pages

### LINKED ISSUE #4449: [Bug] Fix grammar on en & en-us About pages
The following sentences are using grammar/English that could be improved for readability (I have italicised the sections that need fixing):

> FreshRSS is _an RSS feed aggregator to self-host_ like [Kriss Feed](http://tontof.net/kriss/feed/) or [Leed](https://github.com/LeedRSS/Leed). It is light and _easy to take in hand while being powerful and configurable tool_.

> Some design elements come from [Bootstrap](https://twitter.github.io/bootstrap/) although FreshRSS doesn’t use this framework. _[Icons](https://git.gnome.org/browse/gnome-icon-theme-symbolic) come from [GNOME project](https://www.gnome.org/)._ Open Sans font police has been created by [Steve Matteson](https://fonts.google.com/specimen/Open+Sans). FreshRSS is based on [Minz](https://github.com/marienfressinaud/MINZ), a PHP framework.

**Describe the solution you’d like**
A restructuring of the sentences to make them more readable.

### PR DESCRIPTION
**Pull request by DuncanBennie**
Restructure sentences on the en and en-us versions of the About page (index.php) to make them grammatically correct.

Closes #4449

### CODE REVIEW

**Comment by DuncanBennie** [comment]
This is my first PR ever so let me know if I've made any mistakes.

**Review by Frenzie — APPROVED** [review]
Thanks! While we're on the subject, "like Kriss Feed and Leed" seems a bit wordy and unnecessary.

File: app/i18n/en-us/index.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		'bugs_reports' => 'Bug reports',	// IGNORE
 		'credits' => 'Credits',	// IGNORE
 		'credits_content' => 'Some design elements come from <a href="http://twitter.github.io/bootstrap/">Bootstrap</a> although FreshRSS doesn’t use this framework. <a href="https://git.gnome.org/browse/gnome-icon-theme-symbolic">Icons</a> come from the <a href="https://www.gnome.org/">GNOME project</a>. <em>Open Sans</em> font police has been created by <a href="https://fonts.google.com/specimen/Open+Sans">Steve Matteson</a>. FreshRSS is based on <a href="https://framagit.org/marienfressinaud/MINZ">Minz</a>, a PHP framework.',	// IGNORE
-		'freshrss_description' => 'FreshRSS is a self-hostable RSS feed aggregator like <a href="http://tontof.net/kriss/feed/">Kriss Feed</a> or <a href="https://github.com/LeedRSS/Leed">Leed</a>. It is light and easy to use, whilst being a powerful and configurable tool.',	// IGNORE
+		'freshrss_description' => 'FreshRSS is a self-hostable RSS feed aggregator that is light and easy to use whilst being highly powerful and configurable.',	// IGNORE
```
Comment (Frenzie) [inline app/i18n/en-us/index.php]: Btw, whilst is chiefly British but I don't know if that's worth the effort.

Comment (DuncanBennie) [reply app/i18n/en-us/index.php]: > FreshRSS is a self-hostable RSS aggregator and reader. It allows you to read and follow several news websites at a glance without the need to browse from one website to another. FreshRSS is lightweight, configurable, and easy to use.

How about this? I pulled the first two sentences from [the manual](https://freshrss.github.io/FreshRSS/en/).

Otherwise: 
> FreshRSS is a self-hostable RSS feed aggregator that is light and easy to use _while_ being highly powerful and configurable.

Comment (Frenzie) [reply app/i18n/en-us/index.php]: > How about this? I pulled the first two sentences from [the manual](https://freshrss.github.io/FreshRSS/en/).

Yes, I copy edited the manual a bit (fairly superficially, but I smoothed over a fair bit of Frenglish). I completely forgot there even was such a thing as an about page, and that explains my odd feeling that I'd already changed it. ;-)

> Otherwise:

I prefer the first suggestion of course. But yes, _while_ would be the usual way to write that for the en-US localization.

**Comment by DuncanBennie** [comment]
@Frenzie Thanks for your help, I've added another commit. I'm a noob at git and am sure there is a way I could have consolidated the final commit with my two previous commits to make this easier for you, but alas, I don't know how.

**Comment by Frenzie** [comment]
In git, something like `git rebase -i`, squash them together and then you'd have to `git push -f` to force update.

However, on platforms like GitHub and GitLab we can easily "squash and merge" into a single commit so it doesn't matter. In fact it makes it easier to look back at the full history that way, which can sometimes be helpful.

**Comment by Alkarex** [comment]
Thanks!
